src/test/java/gift/controller/api/ItemControllerTest.java

```diff
@@ -42,34 +43,36 @@ class ItemControllerTest {
 
     private String adminToken;
     private String userToken;
-    private Item testItem1;
 
     @BeforeEach
     void setUp() {
         LoginResponse adminLogin = memberService.login(new MemberLoginRequest("admin@example.com", "admin1234"));
         adminToken = adminLogin.token();
-
         LoginResponse userLogin = memberService.login(new MemberLoginRequest("user@example.com", "user1234"));
         userToken = userLogin.token();
     }
 
     @Test
-    @DisplayName("API - 전체 상품 목록 조회 성공 (인증 불필요)")
+    @DisplayName("API - 전체 상품 목록 조회 (페이지네이션 적용)")
     void getAllItems() throws Exception {
-        mockMvc.perform(get("/api/products"))
-            .andExpect(status().isOk());
+        mockMvc.perform(get("/api/products")
+                .param("page", "0")
+                .param("size", "2"))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.content").isArray())
+            .andExpect(jsonPath("$.content", hasSize(2)))
+            .andExpect(jsonPath("$.totalElements").value(3));
```

Comment:
```suggestion
            .andExpect(jsonPath("$.totalElements").value(15));
```
data.sql 을 수정하시면서 테스트가 깨지고 있습니다~